Code-Hills/codehills-be
# Chore ci/cd

# What is this PR
Its integration of circleCI and coverall

# How should be tested
- clone this repo if you don't have
- install dependencies
- and check into Readme you will see badges

# Any Blocker
